scripts/e2etests/VSUtils.ps1

```diff
@@ -207,6 +207,42 @@ function Update-Configuration(
     Start-Process -FilePath $vsFilePath -ArgumentList '/updateConfiguration' -Wait
 }
 
+function UpdateVSInstaller {
+    param(
+        [ValidateSet("16.0")]
+        [string]$VSVersion,
+        [Parameter(Mandatory = $true)]
+        [int]$ProcessExitTimeoutInSeconds
+    )
+
+    $vsMajorVersion = [System.Version]::Parse($VSVersion).Major
+    $vsBootstrapperUrl = "https://aka.ms/vs/$vsMajorVersion/pre/vs_enterprise.exe"
```

Comment:
I noticed Rob was using intpreview and you're using pre. I think preview makes more sense, but there's a more important reason that we should comment on...

```suggestion
    # The Preview channel is intentional since the --update command will update the installer to the latest preview version.  
    # You will run into issues if the machine has a channel that is newer than preview, like IntPreview.
    $vsBootstrapperUrl = "https://aka.ms/vs/$vsMajorVersion/pre/vs_enterprise.exe"
```

Source (Tyler's thread):
> Running “vs_enterprise.exe --update --quiet –wait” will update the installer to the latest preview version. You will run into issues if the machine has a channel that is newer than preview, like IntPreview.

Reply:
Good suggestion! Thanks Donnie.

Reply:
Applied :)